src/tests/JIT/opt/GuardedDevirtualization/enumerablecloning.cs

```diff
@@ -0,0 +1,48 @@
+// Licensed to the .NET Foundation under one or more agreements.
```

Comment:
This is one of those tests where I wish we had the ability to verify that a particular optimization kicked in.